http-clients/apache5-client/src/main/java/software/amazon/awssdk/http/apache5/Apache5HttpClient.java

```diff
@@ -101,14 +101,29 @@
 import software.amazon.awssdk.utils.Validate;
 
 /**
- * An implementation of {@link SdkHttpClient} that uses Apache5 HTTP client to communicate with the service. This is the most
- * powerful synchronous client that adds an extra dependency and additional startup latency in exchange for more functionality,
- * like support for HTTP proxies.
+ * An implementation of {@link SdkHttpClient} that uses Apache HttpClient 5.x to communicate with the service. This is a
+ * full-featured synchronous client that adds an extra dependency and higher startup latency compared to
+ * <a href="https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/http-configuration-url.html">UrlConnectionHttpClient</a>
+ * in exchange for more functionality, like support for HTTP proxies.
  *
- * <p>See software.amazon.awssdk.http.urlconnection.UrlConnectionHttpClient for an alternative implementation.</p>
+ * <p><b>Note:</b> The higher startup latency is compared to UrlConnectionHttpClient, not to the Apache HttpClient 4.5.x
```

Comment:
I don't think this line is necessary. We already state above:

> and higher startup latency compared to UrlConnectionHttpClient

Reply:
Done